scalyr_agent/log_processing.py

```diff
@@ -62,6 +55,12 @@
 from os import listdir
 from os.path import isfile, join
 
+if sys.version_info[0] < (3, 5):
```

Comment:
The `[0]` seems incorrect to me, wouldn't that get you the major version when we want the full tuple?

Reply:
Good catch, thanks.